content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -15,24 +15,33 @@ test: yes
 ---
 
 [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
-describes several ways to configure liveness and readiness probes including:
+describes several ways to configure liveness and readiness probes:
 
 1. [Command](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command)
 1. [HTTP request](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request)
+1. [TCP probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-tcp-liveness-probe)
 
 The command approach works with Istio regardless of whether or not mutual TLS is enabled.
 
 The HTTP request approach, on the other hand, requires special Istio configuration when mutual TLS is enabled.
 This is because the health check requests to the `liveness-http` service are sent by Kubelet,
 which does not have an Istio issued certificate. Therefore when mutual TLS is enabled,
 the health check requests will fail.
```

Comment:
```suggestion
When mutual TLS is enabled, HTTP requests to the `liveness-http` service are sent by the Kubelet,
which does not have an Istio-issued certificate. Therefore, the requests will fail.
```

Reply:
Overall your suggestions look good to me. But this one is a bit broken.

"**When mutual TLS is enabled**, HTTP requests to the `liveness-http` service **are sent by the Kubelet**" This is wrong: the requests are always sent by the Kubelet, but it is only a problem when mTLS is enabled.

How about:
```suggestion
The health check requests to the `liveness-http` service are sent by Kubelet.
This becomes a problem when mutual TLS is enabled, because the Kubelet does not have an Istio issued certificate.
Therefore the health check requests will fail.
```

Reply:
LGTM